src/pages/landing/trade-forex/components/_hero.tsx

```diff
@@ -95,7 +114,7 @@ const HeroComponent = ({ title, content, background_data }) => {
                         {title}
                     </StyledHeader>
                     <HeroContent m="2rem 0 0" direction="column" justify="flex-start">
-                        <Header color="white" type="subtitle-1" weight="normal">
+                        <Header as="h1" color="white" type="subtitle-1" weight="normal">
```

Comment:
Since the `StyledHeader` above defaults to `h1` already. This should be h2 or lower. There should only be one `h1` per page.
```suggestion
                        <Header as="h2" color="white" type="subtitle-1" weight="normal">
```

Reply:
Done. Thanks for the suggestion @sean-binary